notes.go

```diff
@@ -136,7 +136,33 @@ func (srv *notesAPI) DeleteNote(ctx context.Context, req *notesv1.DeleteNoteRequ
 }
 
 func (srv *notesAPI) ListNotes(ctx context.Context, req *notesv1.ListNotesRequest) (*notesv1.ListNotesResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "unimplemented")
+	token, err := srv.authenticate(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	err = validators.ValidateListNoteRequest(req)
```

Comment:
This method has a typo (should be `ValidateListNotesRequest`).  Also it's my fault but there's a bug in that method: `AuthorAccountId` is marked as required but it should be optional (`AuthorAccountId` and `GroupId` should be optional but there should be at least one that is set).